ninja-core/src/test/java/ninja/utils/SimpleClass.java

```diff
@@ -16,6 +16,8 @@
 
 package ninja.utils;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Looks like the link to @objectmapper in the comment has triggered the import by the automatic clean up. Removed it.